packages/oscal-react-library/src/components/OSCALMetadata.tsx

```diff
@@ -306,34 +308,72 @@ const MetadataInfoList: React.FC<MetadataInfoListProps> = (props) => {
   );
 };
 
-export interface OSCALMetadataPartyProps {
+interface OSCALMetadataPartyLogoProps extends ResolvableLinkHref {
+  link: OSCALLink;
+  partyName?: string;
+}
+
+const OSCALMetadataPartyLogo: React.FC<OSCALMetadataPartyLogoProps> = ({
+  link,
+  backMatter,
+  parentUrl,
+  partyName,
+}) => {
+  let logoUri = undefined;
+
+  if (backMatter) {
+    try {
+      logoUri = resolveLinkHref(backMatter, link.href, parentUrl, /^image\//, false);
+    } catch (err) {
+      // Silently fail on unresolved diagram resources
+    }
+  }
+
+  const imageSrc = logoUri ?? link.href;
```

Comment:
```suggestion
  let imageSrc;
  if (backMatter) {
    try {
      imageSrc = resolveLinkHref(backMatter, link.href, parentUrl, /^image\//, false);
    } catch (err) {
      imageSrc = link.href;
    }
  }
```

An idea that may not be entirely correct.

Reply:
You can write the catch as: `catch(_)` if unused param gives trouble or maybe even use `finally` instead.

Reply:
An issue with that is if `backMatter` is falsy, `imageSrc` is never initialized. I had considered doing what you did and then also initializing it on line 323, but that seemed uncessary. I think its kinda a gross chain of logic no matter what, so I chose what I felt was the best. 

I do however need to update that comment...

Reply:
```suggestion
  let imageSrc;
  let logoUri;
  
  if (backMatter) {
    try {
      logoUri = resolveLinkHref(backMatter, link.href, parentUrl, /^image\//, false);
    } finally {
      imageSrc = logoUri ?? link.href;
    }
  }
```
Another idea . . .

Reply:
```suggestion
  let imageSrc;
  let logoUri;
  
  try {
    if (backMatter) {
      logoUri = resolveLinkHref(backMatter, link.href, parentUrl, /^image\//, false);
    }
  } finally {
    imageSrc = logoUri ?? link.href;
  }
```
I'm trying to avoid us having to use half the functionality of a try-catch block and I think we can do that here. This would work in the case of `backMatter` being false.

Reply:
I am not convinced removing the catch is inherently better. I think the fact that we are actively choosing to ignore an error is not something to hide with more complexity. Especially if that means we have to add another `let`, which is also not ideal.

I realize that basing decisions on old code isn't great, but we do this in `OSCALDiagram`. If we decide to do something when `resolveLinkHref` fails, we would go back to the catch in both places.